sys-apps/thunderbolt-software-user-space/metadata.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+	<maintainer type="person">
+		<name>Jonas Licht</name>
+		<email>jonas.licht@fem.tu-ilmenau.de</email>
+	</maintainer>
```

Comment:
Please also add the proxy-maintainer team to this file. See https://wiki.gentoo.org/wiki/Project:Proxy_Maintainers/User_Guide#Adding_a_new_package.